docs/witness_run.md

```diff
@@ -10,7 +10,7 @@ witness run [cmd] [flags]
 
 ```
       --archivista-server string                      URL of the Archivista server to store or retrieve attestations (default "https://archivista.testifysec.io")
-  -a, --attestations strings                          Attestations to record (default [environment,git])
+  -a, --attestations strings                          Attestations to record (default [product,material])
```

Comment:
You'll have to change https://github.com/in-toto/witness/blob/main/options/run.go#L40 and run `make docgen`